### PR TITLE
applications: doc: Update the ipc_radio documentation

### PR DESCRIPTION
**Pull request by dchat-nordic**
Update the ipc_radio documentation with Sysbuild notes and comments from PR-12588.

Jira: NCSIDB-1210

### CODE REVIEW

File: applications/ipc_radio/README.rst [hunk 1]
```diff
@@ -89,24 +90,20 @@ The following files are available:
...
 
-To enable a specific configuration overlay file, use the following command:
-
-.. code-block:: console
-
-   west build |application path| -b board_name -- -DEXTRA_CONF_FILE="overlay-***.conf"
-
-You can add multiple configuration files separated by a semicolon.
-You cannot use :ref:`ble_rpc` together with the HCI :ref:`bluetooth_controller`.
+To enable a specific configuration overlay file please refer to :ref:`building_advanced`.
```
Comment (peknis) [inline applications/ipc_radio/README.rst:103]:
```suggestion
For instructions on how to enable a specific configuration overlay file, see :ref:`building_advanced`.
```

File: applications/ipc_radio/README.rst [hunk 2]
```diff
@@ -47,7 +47,8 @@ The firmware supports the following development kits:
 
 .. table-from-sample-yaml::
 
-To automatically attach the firmware image, you need to use the :ref:`sysbuild`.
+To automatically attach the firmware image, you need to use the :ref:`zephyr:sysbuild`.
+To learn more about :ref:`zephyr:sysbuild` in the nRF Connect for VS Code extension see `nRF Connect Sysbuild`_.
```
Comment (peknis) [inline applications/ipc_radio/README.rst:51]:
```suggestion
To learn more about :ref:`zephyr:sysbuild` in the |nRFVSC| see `nRF Connect Sysbuild`_.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: applications/ipc_radio/README.rst [hunk 3]
```diff
@@ -89,24 +90,20 @@ The following files are available:
 * :file:`overlay-bt_hci_ipc.conf` - Configuration file enabling Bluetooth Low Energy over HCI.
 * :file:`overlay-bt_rpc.conf` - Configuration file enabling Bluetooth Low Energy over RPC.
 
-Building and running
-********************
+.. note::
+   When using Sysbuild, the configuration files are added automatically depending on the Sysbuild configuration.
```
Comment (greg-fer) [inline applications/ipc_radio/README.rst:94]:
```suggestion
   When using sysbuild, the configuration files are added automatically depending on the sysbuild configuration.
```
Global edit for the page: `sysbuild`. Zephyr docs love decapitalized proper names (`west`, not `West`), so we need to follow suit: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/build/sysbuild/index.html -- only the title is capitalized

Comment (greg-fer) [inline applications/ipc_radio/README.rst:94]: I'm having trouble understand this sentence, which previously was featured lower in the file.

I was wondering about "When using Sysbuild, the configuration files are added automatically" before (aren't all added automatically when you generate a build configuration?), but now the addition of "depending on the sysbuild configuration" makes me even more confused.

Real-world example of how I read this sentence: "When you use the light switch, the bulb is turned on automatically depending on the light switch configuration".

What does "depending on the sysbuild configuration" add here? Sounds a lot like a tautology...

File: applications/ipc_radio/README.rst [hunk 4]
```diff
@@ -47,7 +47,8 @@ The firmware supports the following development kits:
 
 .. table-from-sample-yaml::
 
-To automatically attach the firmware image, you need to use the :ref:`sysbuild`.
+To automatically attach the firmware image, you need to use the :ref:`zephyr:sysbuild`.
```
Comment (greg-fer) [inline applications/ipc_radio/README.rst:50]:
```suggestion
To automatically attach the firmware image, you need to use Zephyr's :ref:`zephyr:sysbuild`.
```

File: applications/ipc_radio/README.rst (hunk 2 above)
Comment (greg-fer) [reply applications/ipc_radio/README.rst:51]: I'd say we don't need two links to the same page in two sentences next to one another:
```suggestion

The |nRFVSC| supports sysbuild during `build configuration setup <How to build an application_>`_.
To learn more about sysbuild in the extention, see `sysbuild support`_ page in its documentation.
```

I think this mention of nRF VSC makes more sense in Configuration or in Building and running as a single image...

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: applications/ipc_radio/README.rst (hunk 3 above)
Comment (greg-fer) [reply applications/ipc_radio/README.rst:94]: What configuration files are added automatically? Sysbuild ones? Application ones? Are these different than the "sysbuild configuration" later in the sentence?

Comment (dchat-nordic) [reply applications/ipc_radio/README.rst:94]: The application (as in specifically and only application) can be built with ready configurations overlay files overlay-802154 etc.
If the application is built as a single image for network core the built process requires to use those overlay files in command line (or via VSC extension) -DEXTRA_CONF_FILE="overlay-abc"

However, when you build some bluetooth sample for instance peripheral_lbs with ipc_radio as network core image, you need only to select *sysbuild* config for SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC and sysbuild Cmake files automatically configure the ipc_radio subimage to include the proper overlay file (in this case overlay-bt_hci_ipc.conf"

Comment (dchat-nordic) [reply applications/ipc_radio/README.rst:94]: You can build this ipc_radio application as a single program for netcore. In this case you need to manually specify configuration and/or overlays.

You can also use build bluetooth/ieee802154 application and use sysbuild to automatically configure the ipc_radio for you. In that case the overlays are managed by the sysbuild Cmake and Kconfig files.

**Review by peknis — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Comment by dchat-nordic** [comment]
Rebased to update the spurious compliance build failure.

**Pull request merged**